### PR TITLE
2970 Add test duplicate records

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: accounting/huobi/storage/postgres/postgres_test.go [hunk 1]
```diff
@@ -77,4 +77,9 @@ func TestSaveAndGetAccountingRates(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, len(data), 1)
 	assert.Equal(t, testData[0].FieldAmount, data[0].FieldAmount)
+
+	// test database does not stored duplicated records(with the same id)
+	data, err = hdb.GetTradeHistory(timeutil.TimestampMsToTime(1540793585679), timeutil.TimestampMsToTime(1540793586000))
```
Comment (cyberliem) [inline accounting/huobi/storage/postgres/postgres_test.go]: without another call on Update(), obviously the db will remain the same

Comment (halink0803) [reply accounting/huobi/storage/postgres/postgres_test.go]: Actually the first Update call on 11 records with 10 is the same id with different timestamp. I change the fromTime, toTime params. If it stored duplicate records, the Get function with return more than 1 record for that time range.
